lib/adf_web.py

```diff
@@ -695,6 +697,32 @@ def jinja_enumerate(arg):
             #Also check if index page exists for this case:
             index_html_file = \
                 self.__case_web_paths[web_data.case]['website_dir'] / "index.html"
+            print("index_html_file",index_html_file)
+
+            # Create run info web page
+            run_info_md_file = \
+                self.__case_web_paths[web_data.case]['website_dir'] / self.run_info
+            print("run_info_md_file",run_info_md_file)
+
+            # Read the markdown file
+            with open(run_info_md_file, "r", encoding="utf-8") as mdfile:
+                md_text = mdfile.read()
+
+            # Convert markdown to HTML
+            run_info_html = markdown.markdown(md_text)
+            index_title = "AMP Diagnostics Prototype"
```

Comment:
Not directly relevant for this PR, but at what point should we drop the word `Prototype` from the website title?  It feels like the ADF is used in production-like settings now.